customcommands/tmplextensions.go

```diff
@@ -488,6 +497,118 @@ func tmplDBDelById(ctx *templates.Context) interface{} {
 		return "", err
 	}
 }
+func tmplDBDelMultiple(ctx *templates.Context) interface{} {
+	return func(query interface{}, iAmount interface{}, iSkip interface{}) (interface{}, error) {
+		if ctx.IncreaseCheckCallCounterPremium("db_interactions", 10, 50) {
+			return "", templates.ErrTooManyCalls
+		}
+
+		if ctx.IncreaseCheckCallCounterPremium("db_multiple", 2, 10) {
+			return "", templates.ErrTooManyCalls
+		}
+
+		ctx.GS.UserCacheDel(CacheKeyDBLimits)
```

Comment:
do this at the end after we have deleted the entries, as to avoid somehow refreshing the cache with still old values

Reply:
done